### ISSUE TITLE
I/O Validation

### ISSUE THREAD

**Issue by Lomarandil** [buddyd16/theToolbox#5]
I'd love to see some I/O validation added. Right now, it appears that any invalid input is treated as a zero value. Perhaps an error code or highlight would be better?

![image](https://user-images.githubusercontent.com/45415987/167245093-67bed0e8-4ae7-4ede-b49b-4219ba941d50.png)

(point load 2 location and point moment magnitude errors shown)

**Comment by Lomarandil** [comment]
It appears that loads outside the beam cause a crash Ïnternal server error":
![image](https://user-images.githubusercontent.com/45415987/167245828-4cb0c646-cb3c-473f-9de6-c72316bb267b.png)
 (point moment at 25.5m outside the 25m beam)
 
Interestingly, when I hit back to return, the first load of each category (distributed, point, moment) is preserved. But the others are removed, and the units are flipped back to imperial
![image](https://user-images.githubusercontent.com/45415987/167245872-da79deb0-6d01-4b2e-8ced-993d270dc1de.png)

**Comment by buddyd16** [comment]
yeah I/O validation is on my list of required additions for all the tools. Very much appreciate the testing you have done, Thank you!
